README.md

```diff
@@ -36,6 +36,7 @@ You can add rules:
     "cypress/assertion-before-screenshot": "warn",
     "cypress/no-force": "warn",
     "cypress/no-async-tests": "error",
+    "cypress/no-async-before": "error",
```

Comment:
Does this issue also occur in after/afterEach lifecycle hooks? Maybe `no-async-hooks` might be a better rule, if this is the case.

Reply:
I guess so, I think it has the same potential to break your testcase. Should the hooks also include the tests? one to rule them all?

Reply:
I do know that in before and beforeEach the async code might actually break the test, because of the dependency. But there might not be a problem as such in the after because of race conditions